shared/wallets/sep7-confirm/container.tsx

```diff
@@ -19,7 +19,7 @@ const mapDispatchToProps = (dispatch: Container.TypedDispatch) => ({
 })
 
 const Connected = Container.connect(mapStateToProps, mapDispatchToProps, (stateProps, dispatchProps) => {
-  if (stateProps.loading) {
+  if (stateProps.loading || !stateProps.sep7ConfirmInfo /* TODO is this ok*/) {
```

Comment:
This one's okay, though they mean the same thing.  loading can never be true unless sep7ConfirmInfo was null.

Reply:
Huh, something odd going on, since I can remove this change and others in sep7 without seeming to cause new TS errors on this branch.